src/libs/utils.js

```diff
@@ -392,6 +392,13 @@ export const useCancellation = () => {
   return controller.current.signal
 }
 
+export const withCancellation = WrappedComponent => {
```

Comment:
I understand the evolution of the naming of `withCancellation` (which matches our `useCancellation` hook). However, if I were coming in fresh, I wouldn't immediately recognize `withCancellation` as a thing that introduces a prop called `signal`. What would you think about calling this `withCancellationSignal`?

Although the name is longer, usage of this should shrink over time as we have fewer class components. I find the use of `useCancellation` perfectly readable because the correlation between "cancellation" and "signal" is all together on one line:
```
  const signal = Utils.useCancellation()
```

Reply:
Good idea, I'll implement.